x-pack/plugins/observability_solution/slo/server/services/summary_client.ts

```diff
@@ -103,59 +107,54 @@ export class DefaultSummaryClient implements SummaryClient {
             },
           },
         }),
-        ...(timeslicesBudgetingMethodSchema.is(slo.budgetingMethod) && {
-          good: {
-            sum: { field: 'slo.isGoodSlice' },
-          },
-          total: {
-            value_count: { field: 'slo.isGoodSlice' },
-          },
-        }),
-        ...(occurrencesBudgetingMethodSchema.is(slo.budgetingMethod) && {
-          good: { sum: { field: 'slo.numerator' } },
-          total: { sum: { field: 'slo.denominator' } },
-        }),
+        ...buildAggs(slo),
       },
     });
 
-    const good = result.aggregations?.good?.value ?? 0;
-    const total = result.aggregations?.total?.value ?? 0;
     const source = result.aggregations?.last_doc?.hits?.hits?.[0]?._source;
     const groupings = source?.slo?.groupings;
 
-    let sliValue;
-    if (timeslicesBudgetingMethodSchema.is(slo.budgetingMethod)) {
-      const totalSlices = computeTotalSlicesFromDateRange(
-        dateRange,
-        slo.objective.timesliceWindow!
-      );
-
-      sliValue = computeSLI(good, total, totalSlices);
-    } else {
-      sliValue = computeSLI(good, total);
-    }
-
-    const initialErrorBudget = 1 - slo.objective.target;
-    const consumedErrorBudget = sliValue < 0 ? 0 : (1 - sliValue) / initialErrorBudget;
-    const errorBudget = toErrorBudget(
-      initialErrorBudget,
-      consumedErrorBudget,
-      calendarAlignedTimeWindowSchema.is(slo.timeWindow) &&
-        occurrencesBudgetingMethodSchema.is(slo.budgetingMethod)
+    const sliValue = computeSliValue(slo, dateRange, result.aggregations);
+    const errorBudget = computeErrorBudget(slo, sliValue);
+
+    const burnRates = await this.burnRatesClient.calculate(
+      slo,
+      instanceId ?? ALL_VALUE,
+      [
+        { name: '5m', duration: new Duration(5, DurationUnit.Minute) },
+        { name: '1h', duration: new Duration(1, DurationUnit.Hour) },
+        { name: '1d', duration: new Duration(1, DurationUnit.Day) },
+      ],
+      remoteName
     );
 
     return {
       summary: {
         sliValue,
         errorBudget,
         status: computeSummaryStatus(slo.objective, sliValue, errorBudget),
+        fiveMinuteBurnRate: burnRates.find(({ name }) => name === '5m')?.burnRate ?? 0,
```

Comment:
Might be nice to create a helper 

```suggestion
        fiveMinuteBurnRate: getBurnRate('5m'),
```